rust/main/hyperlane-base/src/metrics/agent_metrics.rs

```diff
@@ -206,7 +206,16 @@ impl ChainSpecificMetricsUpdater {
     }
 
     async fn update_block_details(&self) {
-        if let HyperlaneDomain::Unknown { .. } = self.conf.domain {
+        if let HyperlaneDomain::Unknown {
+            domain_id,
+            domain_name,
+            ..
+        } = &self.conf.domain
+        {
+            debug!(
+                domain_id,
+                domain_name, "Unknown domain, skipping chain metrics"
+            );
             return;
         };
         let chain = self.conf.domain.name();
```

Comment:
could also make the `No chain metrics available` log a `debug`, since we wouldn't see the `trace`